src/lib/__tests__/file-test.js

```diff
@@ -324,4 +327,64 @@ describe('lib/file', () => {
             expect(cache.get).to.not.be.called;
         });
     });
+
+    describe('isVeraProtectedFile()', () => {
+        [
+            'some.vera.pdf.html',
+            '.vera.test.html',
+            'blah.vera..html',
+            'another.vera.3.html',
+            'test.vera.html'
+        ].forEach((fileName) => {
+            it('should return true if file is named like a Vera-protected file', () => {
+                expect(isVeraProtectedFile({ name: fileName })).to.be.true;
+            });
+        });
+
+        [
+            'vera.pdf.html',
+            'test.vera1.pdf.html',
+            'blah.vera..htm',
+            'another.verahtml',
+        ].forEach((fileName) => {
+            it('should return false if file is not named like a Vera-protected file', () => {
+                expect(isVeraProtectedFile({ name: fileName })).to.be.false;
+            });
+        });
+    });
+
+    describe('canDownload()', () => {
+        let file;
+        let options;
+
+        beforeEach(() => {
+            file = {
+                is_download_available: false,
+                permissions: {
+                    can_download: false
+                }
+            };
+            options = {
+                showDownload: false
+            };
+        });
+
+        [
+            [false, false, false, false, false],
+            [false, false, false, true, false],
+            [false, false, true, false, false],
+            [false, true, false, false, false],
+            [true, false, false, false, false],
+            [true, true, true, true, true],
+        ].forEach(([isDownloadable, isDownloadEnabled, havePermission, isBrowserSupported, expectedResult]) => {
```

Comment:
I didn't have the [] here at first, but the test still passed since an array for the first param and undefined for all of the others still passed the test...